go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/code-ready/admin-helper v0.0.7
 	github.com/code-ready/clicumber v0.0.0-20210201104241-cecb794bdf9a
 	github.com/code-ready/machine v0.0.0-20210616065635-eff475d32b9a
-	github.com/containers/gvisor-tap-vsock v0.1.1-0.20210816082554-ebd241aab69f
+	github.com/containers/gvisor-tap-vsock v0.3.1-0.20220208081105-763ad719398b
```

Comment:
Can we use this new version with the current bundles, or do we need an updated gvisor-tap-vsock image?

Reply:
@cfergeau we can use current bundle with new version, there is no change in vm command which is part of gvisor-tap-vsock but we do need to update the image sooner than later. This is not blocker for thsi PR.

Reply:
We do need to update the image for this PR? or is this just a general comment about bundle quality?

Reply:
These snc changes are no longer needed, marking this as resolved.